src/popup.js

```diff
@@ -1,37 +1,41 @@
 var total,
-    acc = 0;
+  acc = 0;
 
 function set(id, start, length, noacc) {
-    var x = Math.round(start / total * 300);
+  var x = Math.round(start / total * 300);
 
-    if (!noacc) {
-        acc += length;
-    }
-    document.getElementById(id + 'When').innerHTML = start;
-    document.getElementById(id).innerHTML = length;
-    document.getElementById(id + 'Total').innerHTML = noacc ? '-' : acc;
-    document.getElementById('r-' + id).style.cssText =
-        'background-size:' + Math.round(length / total * 300) + 'px 100%;' +
-        'background-position-x:' + (x >= 300 ? 299 : x) + 'px;';
+  if (!noacc) {
+    acc += length;
+  }
+  document.getElementById(id + 'When').innerHTML = start;
+  document.getElementById(id).innerHTML = length;
+  document.getElementById(id + 'Total').innerHTML = noacc ? '-' : acc;
+  document.getElementById('r-' + id).style.cssText =
+    'background-size:' + Math.round(length / total * 300) + 'px 100%;' +
+    'background-position-x:' + (x >= 300 ? 299 : x) + 'px;';
+}
+
+function setCount(id, count) {
+  document.getElementById(id + 'Count').innerText = count;
 }
 
 chrome.tabs.getSelected(null, function (tab) {
-    chrome.storage.local.get('cache', function(data) {
-        var t = data.cache['tab' + tab.id],
-        start = t.redirectStart == 0 ?t.fetchStart : t.redirectStart;
+  chrome.storage.local.get('cache', function(data) {
+    var t = data.cache['tab' + tab.id];
+    total = t.totalTime;
```

Comment:
I think you should add `var` here or change `;` to `,` on the previous line otherwise `total` var will be global

Reply:
yes you are right,  I missed it is used in the outer scope